src/Request/RequestHandler.php

```diff
@@ -285,28 +352,45 @@ private function getRequestToken(array $parts = [])
             ]
         ];
 
-        // todo: We must sign each request
-
-        $parameters = [ ];
-
+        $parameters = [];
         $tokenData = $this->handleRequest(
             'GET',
             $serviceUrl,
             $options,
-            $parameters
+            $parameters,
+            null
```

Comment:
This is the entry point for the method that the request will handle, right?

I think the _default_ should be null, and `json_decode` should get passed in where needed.

This means that you will be in better control, and you don't need to know what the null is for.

@make-it-git I think this is an important change to make at this time.